database/db.js

```diff
@@ -0,0 +1,42 @@
+// require('dotenv').config();
+const { Client } = require('pg');
+
+const client = new Client({
+  user: 'kallycao',
+  host: 'localhost',
+  database: 'questionsanswersdb',
+  password: '',
+  port: 5432,
+});
+
```

Comment:
Consider using a environment variable using .env or config.js to hide the important information. Not now, but try to implement b4 deploy :)